src/workerd/util/BUILD.bazel

```diff
@@ -61,12 +71,37 @@ wd_cc_library(
     alwayslink = 1,
 )
 
+wd_cc_library(
+    name = "own-util",
+    hdrs = ["own-util.h"],
+    visibility = ["//visibility:public"],
+)
+
+wd_cc_library(
```

Comment:
this target is used a lot. Thanks for extracting.

Reply:
Yep, I was in the neighborhood and recognized the header.